pkg/client/client.go

```diff
@@ -266,3 +284,27 @@ func buildCustomEndpointsMap(customEndpoints []configv1.AWSServiceEndpoint) map[
 
 	return customEndpointsMap
 }
+
+// useCustomCABundle will set up a custom CA bundle in the AWS options if a CA bundle is configured in the
+// kube cloud config.
+func useCustomCABundle(awsOptions *session.Options, ctrlRuntimeClient client.Client) error {
```

Comment:
Fetching this object each reconcile is a little spammy, but this allows users to potentially update the CA bundle day 2 (or whenever) and we don't need to worry about restarting our controller.

Reply:
The resource will be cached locally. The client will not have to reach out to the server on every reconcile.